### ISSUE TITLE
Clarify license

### ISSUE THREAD

**Issue by IceflowRE** [invopop/gobl.xinvoice#20]
Could you please clarify you license?

You released it as 

> Released publicly under the [Apache License Version 2.0](https://github.com/invopop/gobl.xinvoice/blob/main/LICENSE)

The following statement is against the Apache 2.0 License.

> For commercial licenses please contact the [dev team at invopop](mailto:dev@invopop.com).

It is unclear now, under which conditions this library is published now. Especially because the license file **only** includes the Apache License 2.0 text and does not mention a change and prohibited commercial use.

**Comment by samlown** [comment]
Yes, that is a bit confusing! We'll see if we can reword it. The intent was that the project is released under an Apache 2.0 license, but if you need something else you can get in touch.

We're actually planning to deprecate this project soon in favour of invopop/gobl.cii, which is the underlying format, so we'll be sure to improve the wording for there.
